tests/network-tests/package.json

```diff
@@ -16,7 +16,7 @@
   "dependencies": {
     "@apollo/client": "^3.2.5",
     "@joystream/distributor-node-client": "^0.2.0",
-    "@joystream/types": "^0.20.6",
+    "@joystream/types": "^0.20.7",
```

Comment:
Same comment as w.r.t. `@joystream/cli`, it looks strange to not have the `patch` version at all and only update the `minor` one during a new release.

Reply:
I think we should start at `1.0.0` for mainnet for consistency